packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -93,7 +93,9 @@ describe('KeyringController', () => {
   describe('addNewAccount', () => {
     it('should add new account', async () => {
       const { keyringState: currentKeyringMemState, addedAccountAddress } =
-        await keyringController.addNewAccount();
+        await keyringController.addNewAccount(
+          initialState.keyrings[0].accounts.length,
```

Comment:
It would be ideal to have test coverage both with and without the accountCount parameter. Perhaps we can duplicate this test?

Reply:
Good point, I reordered this method's tests into groups and added this case